database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE zn_data.storage (
+  st_id            VARCHAR(36) NOT NULL PRIMARY KEY,
+  st_type          VARCHAR(32) NOT NULL,
+  st_configuration JSONB       NOT NULL
+);
+
+
+CREATE TABLE zn_data.timeline (
+  t_id              UUID                     NOT NULL PRIMARY KEY,
```

Comment:
`tl` is good 👍 and it's consistent with the 2 chars rule (the exception `ets` is due to conflict).